### PR TITLE
Vtcca 3166

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/controllers/detailedvaluationrequest/DvrController.scala [hunk 1]
```diff
@@ -81,7 +81,7 @@ class DvrController @Inject()(
           optDocuments <- dvrCaseManagement.getDvrDocuments(link.uarn, valuationId, link.submissionId)
           backUrl      <- calculateBackLink(propertyLinkSubmissionId, owner)
           startCheckUrl = config.businessRatesCheckUrl(
-            s"property-link/${link.authorisationId}/assessment/$valuationId?propertyLinkSubmissionId=$propertyLinkSubmissionId&uarn=$uarn&dvrCheck=true")
+            s"property-link/${link.authorisationId}/assessment/$valuationId?propertyLinkSubmissionId=$propertyLinkSubmissionId&uarn=$uarn&dvrCheck=true&isOwner=$owner")
```
Comment (reddy-yattapu) [inline app/controllers/detailedvaluationrequest/DvrController.scala:84]: Just me to understand, is it fixing existing bug or added new query parameter "isOwner=$owner"
If new  query parameter "isOwner=$owner", I hope already business rates check can handle this query paramter

**Comment by reddy-yattapu** [comment]
👍

File: app/controllers/detailedvaluationrequest/DvrController.scala (hunk 1 above)
Comment (habeeb01) [reply app/controllers/detailedvaluationrequest/DvrController.scala:84]: good spot Reddy. I think I might not have had the latest master. I will update from master

Comment (habeeb01) [reply app/controllers/detailedvaluationrequest/DvrController.scala:84]: good spot Reddy, that should not have been committed. That code was from when I was trying to fix another bug. Im updating the PR

**Pull request merged**